packages/components/input/src/Input.styles.ts

```diff
@@ -26,15 +26,10 @@ export const inputStyles = cva(
         false: ['h-sz-44'],
       },
       intent: {
-        neutral: [
-          'ring-outline',
-          'hover:ring-outline-high',
-          'focus:ring-outline-high',
-          'disabled:ring-outline',
-        ],
-        success: ['ring-success', 'ring-success'],
-        alert: ['ring-alert', 'ring-alert'],
-        error: ['ring-error', 'ring-error'],
+        neutral: ['border-outline', 'hover:border-outline-high', 'disabled:border-outline'],
```

Comment:
Is it normal that only the neutral intent has its own **disabled** styles?

Reply:
See https://github.com/adevinta/spark/pull/1560#issuecomment-1757364903